packages/js/ai/package.json

```diff
@@ -72,19 +73,11 @@
 		"webpack-cli": "^3.3.12"
 	},
 	"scripts": {
-		"turbo:build": "pnpm run build:js && pnpm run build:css",
-		"turbo:test": "jest --config ./jest.config.json --passWithNoTests",
```

Comment:
I took a stab at cleaning up the scripts here. I think we need very few things here. Since this is a Typescript package meant to be imported into Typescript projects, it seems to me that we don't need any build scripts, nor do I think that we need `turbo`. 🤔

Reply:
I think at least some of this may be expected from the mono-repo tooling. Also, I think we should ensure that standard JS (non-TS) can use the package as well.

I'm currently getting the following errors when running `pnpm run build` in the root of the mono-repo which I think is related:
```
woo-ai:turbo:build: built modules 257 KiB (javascript) 7.4 KiB (css/mini-extract) 1.16 KiB (asset) [built]
woo-ai:turbo:build:   modules by path ../../node_modules/.pnpm/ 101 KiB 20 modules
woo-ai:turbo:build:   modules by path ./ 156 KiB (javascript) 7.4 KiB (css/mini-extract) 1.16 KiB (asset)
woo-ai:turbo:build:     modules by path ./src/ 156 KiB (javascript) 7.4 KiB (css/mini-extract) 2 modules
woo-ai:turbo:build:     asset modules 84 bytes (javascript) 1.16 KiB (asset)
woo-ai:turbo:build:       ./assets/images/icons/magic.svg 42 bytes (javascript) 808 bytes (asset) [built] [code generated]
woo-ai:turbo:build:       ./assets/images/icons/alert.svg 42 bytes (javascript) 376 bytes (asset) [built] [code generated]
woo-ai:turbo:build:   external "React" 42 bytes [built] [code generated]
woo-ai:turbo:build: 
woo-ai:turbo:build: ERROR in ./src/image-background-removal/background-removal-link.tsx 6:0-93
woo-ai:turbo:build: Module not found: Error: Can't resolve '@woocommerce/ai' in '/home/joelt/project/woomono/woocommerce_add-jetpack-jwt-library/plugins/woo-ai/src/image-background-removal'
woo-ai:turbo:build:  @ ./src/image-background-removal/index.tsx 4:0-66 17:39-60
woo-ai:turbo:build:  @ ./src/index.ts 5:0-36
woo-ai:turbo:build: 
woo-ai:turbo:build: ERROR in ./src/image-background-removal/image_utils/uploadImageToLibrary.ts 4:0-54
woo-ai:turbo:build: Module not found: Error: Can't resolve '@woocommerce/ai' in '/home/joelt/project/woomono/woocommerce_add-jetpack-jwt-library/plugins/woo-ai/src/image-background-removal/image_utils'
woo-ai:turbo:build:  @ ./src/image-background-removal/image_utils/index.ts 1:0-39 1:0-39

```